.circleci/config.yml

```diff
@@ -85,7 +87,7 @@ commands:
       - checkout
       - install-rustup
       - install-android-targets
-      - test-setup:
+      - setup-rust-toolchain:
```

Comment:
Guess we should probably first prepare the rust toolchain, then install the android targets
(simple fix: just swap these two tasks)

That was already wrong in the old config. It works because the stable toolchain is already installed anyway, but I'd prefer if this would be semantically correct here.

Reply:
Make sense!